### DISCUSSION TITLE
Future of WinUI support on Linux

### DISCUSSION THREAD

**Discussion by Danie-Brink** [unoplatform/uno#5909] [General]
Will uno eventually support WinUI on Linux? The previews seem to be only Win/iOS/Android/macOS/WebAssembly

**Comment by jeromelaban** [comment]
Uno already supports Linux, using WinUI. See [this article](https://platform.uno/docs/articles/get-started-dotnet-new.html#uno-platform-blank-application-for-winui-30---preview) to get started.

**Reply by Danie-Brink** [reply]
Thank you. I did look at that but the section on WebUI only lists iOS, Android, WebAssembly and macOS so wasn't sure.
